config/config.go

```diff
@@ -1391,7 +1410,8 @@ type RemoteWriteConfig struct {
 
 	// We cannot do proper Go type embedding below as the parser will then parse
 	// values arbitrarily into the overflow maps of further-down types.
-	HTTPClientConfig HTTPClientConfig `yaml:",inline"`
+	HTTPClientConfig   HTTPClientConfig   `yaml:",inline"`
+	QueueManagerConfig QueueManagerConfig `yaml:",inline"`
```

Comment:
As we inline the `HTTPClientConfig` in other places I guess it's fine, but I think this config should not be. Wdyt @brian-brazil ?